### PR TITLE
fix(header): Added getting-started header and logos

### PR DESCRIPTION
**Pull request by dlabrecq**
Added getting-started header and OpenShift logos per latest mockups.

FYI, @jryhanyc @catrobson

screen shot 2017-04-21 at 4 22 29 pm

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a68d446e357de72256aa71807042ebf0ae96014b
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by fabric8cd** [comment]
@dlabrecq snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-979-fabric8-ui.int.rdu2c.fabric8.io

File: src/app/header/header.component.html [hunk 1]
```diff
@@ -1,144 +1,151 @@
...
-      </li>
-    </ul>
-  </div>
-  <div class="collapse navbar-collapse navbar-collapse-1">
-    <ul class="nav navbar-nav navbar-primary persistent-secondary navbar-left">
-      <!-- This part of the menu is dynamic based on context -->
-      <li class="context dropdown recent-items-dropdown" dropdown *ngIf="loggedInUser">
-        <a class="recent-items-toggle" dropdown-open data-toggle="tooltip" title="{{context.name}}">
-          <div id="header_dropdownToggle" class="recent-item-text">
+              <span>{{loggedInUser.attributes.fullName}}</span>
```
Comment (joshuawilson) [inline src/app/header/header.component.html]: Why did you indent this and everything below it? I looks like it shouldn't have changed.

Comment (dlabrecq) [inline src/app/header/header.component.html]: When I cut and pasted the content into the template tag, this was automatically indented. I thought we would want things properly indented, but can back it out?

Comment (joshuawilson) [inline src/app/header/header.component.html]: OH, it is the span above that is not aligned. Got it. Thanks.

**Pull request merged**